test/functional/inspec_exec_test.rb

```diff
@@ -214,6 +214,15 @@ def stderr
     assert_exit_code 0, out
   end
 
+  it "can execute when control namespace clashes with input" do
+    inspec("exec " + namespace_clash_control + " --no-create-lockfile")
+
+    _(stdout).wont_include("Control Source Code Error")
+    _(stdout).must_include "\nProfile Summary: 1 successful control, 0 control failures, 0 controls skipped\n"
+    _(stderr).must_equal ""
+    assert_exit_code 0, out
+  end
```

Comment:
This might be better in test/functional/waivers_test, as it is a fix for a regression related to the waivers system. As it is a regression test might also add a link to the issue.

Reply:
Great point, I wondered if I could house this in a better location. I'll move it and link to the regression issue.